source/includes/administration/_knowledge_base.md

```diff
@@ -0,0 +1,270 @@
+                {
+                  "language": "fr",
+                  "id": "83297d87-e984-42b8-9356-2edef188b6c5",
+                  "text": "Gestion des modèles",
+                  "type": "title",
+                  "version": 1
+                },
+                {
+                  "language": "fr",
+                  "id": "07bec89a-870e-4506-9d45-8ea977b2f9f2",
+                  "text": "gestion-des-modeles",
+                  "type": "url_slug",
+                  "version": 1
+                },
+                {
+                  "language": "fr",
+                  "id": "aa2deeee-39a6-40f8-886b-1779c8bdbfa0",
+                  "text": "\n\n### Créer un modèle à partir d'une instance existante...",
+                  "type": "body",
+                  "version": 2
+                }
+              ],
+              "rank": 9,
+              "id": "05353b4b-5675-4631-b91a-6872f385c3b5",
+              "published": true,
+              "gitKey": "cloudstack-compute-service/working-with-instance-templates",
+              "updatedAt": "2020-09-24T15:54:36.000Z"
+            }
+          ],
+          "updatedAt": "2020-09-24T15:54:34.000Z"
+        }
+      ]
+    }
+  ]
+}
+```
+
+List the knowledge base configured on the system.
+
+Query Params | &nbsp;
+---- | -----------
+`organization_id`<br/>*UUID* | Return only knowledge base for the provided organization id. If not provided, will default to the user organization.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the knowledge base.
+`organization.id`<br/>*UUID* | The organization id that the knowledge base is linked to.
+`state`<br/>*String* | The source state. Possible values:  IMPORTING, SYNCHING, READY, ERROR.
+`sources`<br/>*Array[Object]* | List of sources associated to the knowledge base
+`sources.id`<br/>*UUID* | The source id.
+`sources.url`<br/>*string* | The source URL.
+`sources.branch`<br/>*string* | The source branch.
+`sources.createdDate`<br/>*String* | The date the source was created.
+`sources.lastSyncDate`<br/>*String* | The last date the source was synchronized.
+`categories.id`<br/>*UUID* | The category id.
+`categories.gitKey`<br/>*string* | The category git key.
+`categories.rank`<br/>*integer* | The category rank.
+`categories.icon`<br/>*string* | The category icon.
+`categories.createdAt`<br/>*string* | The creation date.
+`categories.updatedAt`<br/>*string* | The update date.
+`categories.featured`<br/>*boolean* | If the category needs to be displayed.
+`categories.translations`<br/>*Array[Object]* | The translation objects for the category.
+`categories.translations.id`<br/>*UUID* | The id of the translation.
+`categories.translations.language`<br/>*string* | The language of the translation.
+`categories.translations.text`<br/>*string* | The content of the translation.
+`categories.translations.type`<br/>*string* | The type of translation. Possible values are: title, body, description or url_slug.
```

Comment:
Category translations can only be of type "title"

Reply:
Unless this isn't for notification category?

Reply:
it is not the same as the notification category, but put use the proper values

Reply:
2e431cc0d0e01031facb1b80afb966f6b042c0b7